apps/services/user-profile/src/config.ts

```diff
@@ -19,10 +18,6 @@ export const UserProfileConfig = defineConfig({
       migrationDate: new Date(
         env.optional('USER_PROFILE_MIGRATION_DATE') ?? '2024-04-12',
       ),
```

Comment:
Should this be removed as well?

Reply:
No, we use this check if we are able to share the data. We check if last nudge is newer than this date to know if the user has seen the new screens due to consent statement on them.